src/tool_help.c

```diff
@@ -332,6 +332,8 @@ static const struct helptxt helptext[] = {
    "TLS 1.3 proxy cipher suites"},
   {"    --proxy-tlsauthtype <type>",
    "TLS authentication type for HTTPS proxy"},
+  {"    --tls-use-session-tickets",
+   "Enable TLS session tickets (NSS)"},
```

Comment:
note that this list is generated (see comment above the list), so the contents need to be in files in `docs/cmdline-opts/` - you need to create a new file there for a new option

Reply:
ok I added a file tls-sessiontickets.d should I drop this two lines then?

Reply:
No, they still need to be there. but my comment about "(NSS)" applies here too...

Reply:
Missed you comment according (NSS) So I dropped it. But now its not clear that RFC 5077 is only supported by NSS implementation of TLS or I'am wrong here?